### PR TITLE
deep_ae fix

### PR DESCRIPTION
**Pull request by 1vn**
using thin dense sigmoid layers, remove weight_reg

also use tmp instead of /tmp for trianing

### CODE REVIEW

File: models/deep_ae.py [hunk 1]
```diff
@@ -3,6 +3,8 @@
 
 def get_autoencoder(params):
   """Callable model function compatible with Experiment API."""
+  n_hidden = 50
```
Comment (aidangomez) [inline models/deep_ae.py]: These should be parameters, not hard-coded. Maybe make params.enc_layers = [50, 50] and params.enc_layers = [50, 784] and then you can iterate over the enc_layers and construct them from those width values.

File: models/deep_ae.py [hunk 2]
```diff
@@ -11,28 +13,26 @@ def autoencoder(features, labels, mode):
...
       ed = params.encoder_depth
       for ei in range(ed):
-        tier_width = (
-            (ed - ei - 1) * data_size + ei * params.hidden_size) // (ed - 1)
+        tier_width = n_hidden
         x = tf.layers.dense(
             x,
             tier_width,
-            activation=tf.nn.relu,
+            activation=tf.nn.sigmoid,
```
Comment (aidangomez) [inline models/deep_ae.py]: Let's test if sigmoid is necessary, see if you can swap to relu.

File: models/deep_ae.py [hunk 3]
```diff
@@ -41,11 +41,12 @@ def autoencoder(features, labels, mode):
       tf.summary.image("original",
                        tf.reshape(labels, [params.batch_size, 28, 28, 1]))
 
-      mse_loss = tf.losses.mean_squared_error(labels, x)
-      weight_reg = tf.add_n(
-          [tf.nn.l2_loss(w) for w in tf.trainable_variables()])
-      tf.summary.scalar("reconstr_err", mse_loss)
-      total_loss = mse_loss + weight_reg
+      total_loss = tf.losses.mean_squared_error(labels, x)
+      # weight_reg = tf.add_n(
```
Comment (aidangomez) [inline models/deep_ae.py]: Don't commit commented code

File: models/s2s_lstm.py [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import tensorflow as tf
+
+
+def get_s2s_lstm(params):
```
Comment (aidangomez) [inline models/s2s_lstm.py]: This shouldn't be here

File: models/registry.py [hunk 5]
```diff
@@ -1,11 +1,13 @@
 from .deep_ae import get_autoencoder
 from .nestdrop_ae import get_nestdrop_autoencoder
 from .act_ae import get_act_autoencoder
+from .s2s_lstm import get_s2s_lstm
```
Comment (aidangomez) [inline models/registry.py]: This shouldn't be here

File: models/registry.py [hunk 6]
```diff
@@ -1,11 +1,13 @@
...
 from .vae import get_vae
 
 _MODELS = {
     "ae": get_autoencoder,
     "ndae": get_nestdrop_autoencoder,
     "actae": get_act_autoencoder,
     "vae": get_vae,
+    "s2s": get_s2s_lstm,
```
Comment (aidangomez) [inline models/registry.py]: This shouldn't be here

File: models/deep_ae.py [hunk 7]
```diff
@@ -41,11 +41,12 @@ def autoencoder(features, labels, mode):
       tf.summary.image("original",
                        tf.reshape(labels, [params.batch_size, 28, 28, 1]))
 
-      mse_loss = tf.losses.mean_squared_error(labels, x)
-      weight_reg = tf.add_n(
-          [tf.nn.l2_loss(w) for w in tf.trainable_variables()])
-      tf.summary.scalar("reconstr_err", mse_loss)
-      total_loss = mse_loss + weight_reg
```
Comment (aidangomez) [inline models/deep_ae.py]: change this line to `total_loss = mse_loss + params.weight_reg * weight_reg` and create a parameter that defaults to zero for `weight_reg`

File: hparams/registry.py [hunk 8]
```diff
@@ -26,8 +30,6 @@
 vae = ae_default
 vae.batch_size = 64
 vae.hidden_size = 100
-vae.encoder_depth = 2
```
Comment (aidangomez) [inline hparams/registry.py]: vae needs these variables so you'll need to .add_hparam then

File: hparams/registry.py [hunk 9]
```diff
@@ -34,5 +34,7 @@
 vae.learning_rate_decay_interval = 10000
 vae.learning_rate_decay_rate = 0.1
 vae.add_hparam("l2_loss", 0.0)
+vae.add_hparam("encoder_depth", 2)
```
Comment (aidangomez) [inline hparams/registry.py]: Actually, had another thought; can you update the code for vae, actae, and ndae so that they use the same `enc_layers` thing as the simple_ae?
